YouVsVirus/Assets/Scripts/levelgethome/CreatePopLevelgethome.cs

```diff
@@ -86,10 +92,19 @@ void Start()
         private void CreateHumans()
         {
             // place player near top left edge of the screen (using the not shrunk screen bounds)
-            Vector3 coords = new Vector3(-randomGridForHumans.screenBounds.x / 0.8f + 0.2f, randomGridForHumans.screenBounds.y / 0.8f - 0.2f, 0f);
+            Vector2 coords;
+            
+            // Create random coordinates and check whether they are spawnable
+            do
+            {
+                coords = new Vector2(-randomGridForHumans.screenBounds.x / 0.8f + 0.2f, randomGridForHumans.screenBounds.y / 0.8f - 0.2f);
```

Comment:
I don't really understand this: You write // Create random coordinates and check whether they are spawnable. But his is not the creation of random coords, just one coordinate that is used to place the player on the screen. It only uses the screenBounds from the game object random grid

Reply:
Ah, ok, old file?